spec/classes/puppet_agent_prepare_stringify_facts_spec.rb

```diff
@@ -1,49 +0,0 @@
-require 'spec_helper'
-
-describe 'puppet_agent::prepare::stringify_facts' do
-  context 'supported operating system families' do
-    %w[Debian RedHat SuSE].each do |osfamily|
-      case osfamily
-      when 'SuSE'
-        os = 'SLES'
-        osmajor = '10'
-      else
-        os = 'foo'
-        osmajor = '42'
-      end
-
-      facts = {
-        operatingsystem: os,
-        operatingsystemmajrelease: osmajor,
-        architecture: 'bar',
-        osfamily: osfamily,
-        lsbdistid: osfamily,
-        lsbdistcodename: 'baz',
-        mco_server_config: nil,
-        mco_client_config: nil
-      }
-
-      context "on #{osfamily}" do
-        if Puppet.version < '4.0.0'
-          context 'on Puppet 3 or lower' do
-            it {
-              is_expected.to contain_ini_setting('puppet stringify_facts').with(
-                ensure: 'present',
-                value: false
-              )
-            }
-          end
-        else
-          context 'on Puppet 4 or higher' do
```

Comment:
Why's this test removed?

Reply:
I can't actually find stringify_facts at all anymore in the puppet 4+ docs. [These deprecation notes for 3](https://puppet.com/docs/puppet/3.8/deprecated_settings.html#stringifyfacts--true) suggest that it's still possible to set it to true there, though, so I will work on restoring this unless someone else has more info.